include/wx/localedefs.h

```diff
@@ -89,6 +89,19 @@ enum wxLocaleInfo
     // the character used as decimal point (for wxLOCALE_CAT_NUMBER or MONEY)
     wxLOCALE_DECIMAL_POINT,
 
+    // check whether locale uses the metric system for measurements
+    // returns "Yes", "No", or "Unknown"
+    wxLOCALE_MEASURE_METRIC,
+
+    // the currency symbol (for example "$")
+    wxLOCALE_CURRENCY_SYMBOL,
+
+    // the currency ISO code (for example "USD")
+    wxLOCALE_CURRENCY_CODE,
+
+    // the number of currency digits
+    wxLOCALE_CURRENCY_DIGITS,
```

Comment:
We could also define `wxLocaleCurrencyInfo` struct with `int digits` field which would avoid to have to return this information as a string.